orcestra_book/plans/HALO-20240903a.md

```diff
@@ -71,55 +71,59 @@ atr_radius = 38e3*1.852
 c_atr_nw = LatLon(17.433,-23.500, label = "c_atr")
 c_atr_se = LatLon(16.080,-21.715, label = "c_atr")
 
+c_new = LatLon(15.580,-22.1, label = "c_atr")
 # Define dates for flight
 
 flight_time   = datetime(2024, 9, 3, 12, 0, 0)
 flight_index = f"HALO-{flight_time.strftime('%Y%m%d')}a"
 
 # Load satellite tracks 
 
-ec_fcst_time  = "2024-09-01"
-ec_track = orcestra.sat.SattrackLoader("EARTHCARE", ec_fcst_time, kind="PRE") \
-    .get_track_for_day(f"{flight_time:%Y-%m-%d}")\
-    .sel(time=slice(f"{flight_time:%Y-%m-%d} 14:00", None))
+if (True):
+    ec_fcst_time  = "2024-09-01"
+    ec_track = orcestra.sat.SattrackLoader("EARTHCARE", ec_fcst_time, kind="PRE") \
+        .get_track_for_day(f"{flight_time:%Y-%m-%d}")\
+        .sel(time=slice(f"{flight_time:%Y-%m-%d} 14:00", None))
 
 # Create elements of track
 
-ec_north = point_on_track(ec_track,lat= 15.50).assign(label = "ec_north") 
+ec_north = point_on_track(ec_track,lat= 14.75).assign(label = "ec_north") 
 ec_south = point_on_track(ec_track,lat=  2.00).assign(label = "ec_south")
 c_north  = point_on_track(ec_track,lat= 13.00).assign(label = "c_north")
 c_south  = point_on_track(ec_track,lat=  4.00).assign(label = "c_south")
 c_mid    = point_on_track(ec_track,lat=c_south.towards(c_north).lat).assign(label = "c_mid")
-c_atr    = LatLon(17.5,-24.00, label = "c_atr")
+c_atr    = LatLon(ec_north.lat+atr_radius/1852./60.,-22.1, label = "c_atr")
+j_beg    = LatLon(lat=ec_north.lat,lon=-25.).assign(label = "junction")
+j_end    = LatLon(lat=ec_north.lat,lon=sal.lon).assign(label = "junction")
```

Comment:
There are two points which carry the same label. This is currently not something which is caught automatically (it likely should), but it can confuse people down stream, as this results in mutliple waypoints which are called the same, but have different coordinates.